python/GafferDispatchTest/DispatcherTest.py

```diff
@@ -207,7 +213,19 @@ def onlyRunOnce( dispatcher, nodes ) :
 
 			return False
 
-		connection = GafferDispatch.Dispatcher.preDispatchSignal().connect( onlyRunOnce )
+		class CapturingSlot2( list ) :
```

Comment:
I'm confused about why we need this rather than just a regular `CapturingSlot`; I tried here with a regular one and all dispatch tests passed fine.

Reply:
Addressed in d1ae63d3a0ca14fdb382e920a6e32980829f3cc7